<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFriends</class>
 <widget class="QWidget" name="FormFriends">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>536</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ebrima</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SteamAchievementsStatistic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GroupBoxFilter">
     <property name="title">
      <string>      Фильтр</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="LineEditName">
          <property name="placeholderText">
           <string>Найти друга</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonFind">
          <property name="text">
           <string>  Поиск</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="ComboBoxStatus">
          <property name="currentText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxOpenProfile">
          <property name="text">
           <string>Публичный профиль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CheckBoxFavorites">
          <property name="text">
           <string>Избранные</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="FrameSelected">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="LabelFriendIcon">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelFriendName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelFriendStatus">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelFriendPublic">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonFriendGoTo">
        <property name="toolTip">
         <string>Перейти на профиль</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonFriendFavorite">
        <property name="toolTip">
         <string>Добавить в избранное</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="TableWidgetFriends">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>8</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>33</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>44</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Иконка</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Новый столбец</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Статус</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Приватность</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">айди</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Перейти на профиль</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Избранное</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>243</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageFirst">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelBackDots">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageBack2">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageBack">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageNow">
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageNext">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageNext2">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabelNextDots">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPageLast">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>242</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
